Tasks/SqlAzureDacpacDeployment/task.json

```diff
@@ -54,7 +54,7 @@
 			"required": true,
 			"groupName": "source",
 			"defaultValue": "",
-			"helpMarkDown": "Location of the DACPAC file on the automation agent or on a UNC path like \\BudgetIT\\Web\\Deploy\\FabrikamDB.dacpac. Predefined variables like $(Build.StagingDirectory) can be also used"
```

Comment:
You have to have a format \\\\\\\\BudgetIT\\WebDeploy\\WebDeployPackage.zip to show \\ in UI. In this path you have Web\Deploy you might want to club